training-service/src/app/pages/content/content.component.css

```diff
@@ -1,17 +1,21 @@
+
+
+
 .col-sm img {
   width: 80%;
   padding-top: 1rem;
 }
 
-h3 {
-  padding: 2.5rem 0rem 2.5rem 0rem;
+p {
```

Comment:
Do not add styles directly to html elements, 
always create class and add class to elem.